test/fixtures/s3-fixture/src/test/java/fixture/s3/S3HttpHandlerTests.java

```diff
@@ -83,6 +83,54 @@ public void testSimpleObjectOperations() {
         );
     }
 
+    public void testGetWithBytesRange() {
```

Comment:
I wonder if we should move this test into the repository tests?

Reply:
This method tests the behavior of the fixture, which is important IMO. 

I added 3rd party tests in #106239 for bytes range requests on Azure/GCS/S3.